frontend/public/locales/en/translation.json

```diff
@@ -494,4 +494,4 @@
       }
     }
   }
-}
+}
```

Comment:
Not sure what happened here. It is usually prefered to leave an empty line at the end of files.